app/src/main/res/values/strings.xml

```diff
@@ -1,14 +1,34 @@
 <resources>
-    <string name="app_name">NerdsExcercising</string>
+    <string name="app_name">invision</string>
     <!-- Strings related to login -->
+    <string name="title">invision</string>
+    <string name="subtitle">Track results, complete challenges, set goals. All in one place.</string>
     <string name="prompt_email">Email</string>
     <string name="prompt_password">Password</string>
-    <string name="action_sign_in">Sign in or register</string>
+    <string name="action_sign_in">Login with Google</string>
     <string name="action_sign_in_short">Sign in</string>
+    <string name="sign_up">Not a member? Sign Up</string>
+    <string name="welcome_image_desc">Welcome Page Illustration</string>
     <string name="welcome">"Welcome !"</string>
     <string name="invalid_username">Not a valid username</string>
     <string name="invalid_password">Password must be >5 characters</string>
     <string name="login_failed">"Login failed"</string>
+    <!--  Strings for home page  -->
+    <string name="home_header_desc">"The background for home page header."</string>
+    <string name="hello_user">Hey, Towa</string>
+    <string name="home_tagline">Let\'s exercise</string>
```

Comment:
![Capture](https://user-images.githubusercontent.com/25514702/154826246-e5b3f6b8-cac5-42be-9097-05894c53710f.PNG)
No need to escape it :)

Reply:
nevermind, there's no escape character on the phone